### DISCUSSION TITLE
Remix without client-side React

### DISCUSSION THREAD

**Discussion by tavoyne** [remix-run/remix#3357] [Q&A] [answered]
Is it possible to use Remix but without having React as part of the client bundle? In other words, no hydration.

**Comment by Xiphe** [comment, marked as answer]
You can get rid of the whole JS client bundle by removing [`<Scripts />`](https://remix.run/docs/en/v1/api/remix#links-livereload-meta-scripts-scrollrestoration) in your `app/root.tsx` thereby making your app server side only. 

After that you still can add other script tags for micro-enhancements later on or re-enable the client.

**Reply by tavoyne** [reply]
Thank you

**Reply by Xiphe** [reply]
see https://remix.run/docs/en/v1/guides/disabling-javascript
